src/main.zig

```diff
@@ -162,6 +162,13 @@ pub fn main() anyerror!void {
         return mainArgs(gpa_tracy.allocator(), arena, args);
     }
 
+    // On WASI, the CWD and absolute paths are emulated w.r.t. the available preopens.
+    // Here we initialize that machinery and set the CWD to ".", where the user is
+    // expected to map their current directory using "--dir=."
```

Comment:
Reordering the sentence makes it more understandable:
"WASI: The --dir= option instructs the web assmebly runtime to preopen a directory, and make it available to the program as a capability which can be used to open files inside that directory. The CWD and absolute paths are emulated w.r.t. the available preopens." [source](https://github.com/bytecodealliance/wasmtime/blob/main/docs/WASI-tutorial.md)